details/CLUSTER_DEFINITION.md

```diff
@@ -62,15 +64,17 @@ This example here shows a cluster definition in JSON format, as it may be submit
     "workers": [
         {
             "aws": {
-              "instance_type": "m3.large"
+                "instance_type": "m3.large"
+                "docker_volume_size": {"size_gb": 100},
             },
             "labels": {
                 "nickname": "first node"
             }
         },
         {
             "aws": {
-              "instance_type": "m3.large"
+                "instance_type": "m3.large"
```

Comment:
missing a comma at the end of the line here.

Reply:
actually I've put the `docker_volume_size` into the provider specific configuration and thats wrong. fixed it.